docs/features/user-watchlists.md

```diff
@@ -40,21 +40,21 @@ For logged in users, the favorites and watch history are stored server side at t
 
 To ensure a **cross-device experience**, we standardize on the following dataformat: 
 
-### Watch history format
+### Continue Watching format
```

Comment:
This is actually 'Watch history', which we might also use for 'Because you watched' and related functionality (in future)